indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -206,8 +208,10 @@ def _send_emails(self, form):
             template = get_template_module('events/registration/emails/custom_email.html',
                                            email_subject=form.subject.data, email_body=email_body)
             bcc = [session.user.email] if form.copy_for_sender.data else []
+            attachments = get_ticket_attachment(registration) if form.attach_ticket.data else None
```

Comment:
i think this might fail when ticketing is not enabled since you remove the field from the form. if it does fail, you need to check `'attach_ticket' in form and form.attach_ticket.data`